galasa-managers-parent/galasa-managers-testingtools-parent/dev.galasa.selenium.manager.ivt/src/main/java/dev/galasa/selenium/manager/ivt/SeleniumManagerIVT.java

```diff
@@ -0,0 +1,62 @@
+package dev.galasa.selenium.manager.ivt;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.apache.commons.logging.Log;
+
+import dev.galasa.Test;
+import dev.galasa.core.manager.Logger;
+import dev.galasa.selenium.ISeleniumManager;
+import dev.galasa.selenium.IWebPage;
+import dev.galasa.selenium.SeleniumManager;
+import dev.galasa.selenium.SeleniumManagerException;
+
+@Test
+public class SeleniumManagerIVT {
+
+    @Logger
+    public Log logger;
+
+    @SeleniumManager
+    public ISeleniumManager seleniumManager;
+
+    @Test
+    public void sendingKeysAndClearingFields() throws SeleniumManagerException {
+        IWebPage page = seleniumManager.allocateWebPage("https://duckduckgo.com", "A");
```

Comment:
Would it not be preferable to test against galasa.dev?

Reply:
It would be great to test against galasa.dev, however the selenium web driver cannot see the IBM cookie overlay which pops up when the page has loaded. I have retrieved the page source from selenium and it sees the overlay making the page uninteractable, but not the cookie box itself.